### DISCUSSION TITLE
Allow using an external editor

### DISCUSSION THREAD

**Discussion by 12people** [standardnotes/forum#2688] [Suggestions]
**Is your feature request related to a problem? Please describe.**
While Standard Notes offers a variety of editors, I find that they're all missing core features like in-note search functionality with navigation between items. I have gotten comfortable with using certain external Markdown editors and would like to continue using them when I can.

**Describe the solution you'd like**
It'd be great to be able to open a note inside an external editor, just like Joplin's  "Toggle external editing" feature.

**Comment by ArtistHB** [comment]
This is probably a simple question, but how do you make the font bigger in Standard Notes?
I can't figure it out or find it in the help pages. I've tried the show Bold editor which has a change font button, but it doesn't work for the note in question. I'm not sure where that setting is applied but it's not helping my old eyes read the note I'm editing.  I've paid for the extended subscription, so I'm thinking I should have that feature somewhere.  thx for any help, sorry for the branch question in this forum. I'm on a Mac.
